scripts/gen-proto.sh

```diff
@@ -14,4 +15,6 @@ buf generate
 cp -r $GEN_DIR/rollup/v1/* $ROLLUP_DIR/types
 rm -rf $GEN_DIR/rollup/v1
 
-# TODO: move the testapp module message types to the testapp/x/testmodule module
+# move the generated testapp module message types to the testapp/x/testmodule module
+cp -r $GEN_DIR/testapp/v1/* $TESTMODULE_DIR/types
+rm -rf $GEN_DIR/testapp/v1
```

Comment:
Can we remove the entire `$GEN_DIR` now?

Reply:
The ex-pulsar `module.pb.go` files are still generated there, however I felt like they didn't really make sense to include in the module itself. The Cosmos SDK repo includes their generated `module.pulsar.go` files in a separate `api` directory, would renaming the `gen` directory to `api` make sense here as well?

Reply:
Thanks, I forgot about those... let's leave it for now.